DOCS/man/options.rst

```diff
@@ -5700,6 +5700,31 @@ them.
     results, as can missing or incorrect display FPS information (see
     ``--display-fps-override``).
 
+``--egl-config-id=<ID>``
+    (EGL only)
+    Select EGLConfig with specific EGL_CONFIG_ID.
+    Rendering surfaces and contexts will be created using this EGLConfig.
+    You can use ``--msg-level=vo=trace`` to obtain a list of available configs.
+
+``--egl-output-format=<auto|rgb8|rgb10|rgba8|rgb10_a2|rgb16f|rgba16f>``
+    (EGL only)
+    Select a specific EGL output format to utilize for OpenGL rendering.
+    This option is mutually exclusive with ``--egl-config-id``.
+    "auto" is the default, which will pick the first usable config
+    based on the order given by the driver.
```

Comment:
```suggestion
    with at least 32-bit color based on the order given by the driver.
```

Reply:
I use the expression "first usable config" not "first available config", which means the first config that can be used by mpv. I think it implies "at least 32-bit color". Explicitly point that out may be a little wordy.

Reply:
Yeah I wasn't sure if it had to be this specific either. You can keep it like before.